fendermint/app/options/src/lib.rs

```diff
@@ -75,6 +75,10 @@ pub struct Options {
     )]
     pub home_dir: PathBuf,
 
+    /// Set a custom directory for ipc log files.
+    #[arg(short = 'd', long, default_value = "/tmp/ipc/logs", env = "FM_LOG_DIR")]
```

Comment:
Why not put these relative to the home directory into a `logs` sub-directory, e.g. `~/.fendermint/logs`?

Alternatively `/var/log` is the standard place.

I'd also avoid the `ipc` name, stick to `fendermint`.

Reply:
removed default value, see below comment.